### DISCUSSION TITLE
I suggest a robot that allows us to do what we want in the team talk program and that improves things the way we want them.

### DISCUSSION THREAD

**Discussion by alessio0292** [BearWare/TeamTalk5#1343] [Ideas]
I suggest a robot that allows us to do what we want in the team talk program and that improves things the way we want them.
Dear Bjoern @bear101, @CoBC and all,
Please create a robot that allows us to do what we want and that improves things in the team talk program the way we want them for example: If we want the ogg vorbis encoder, he, the robot, has to put it.
Greetings,
Alessio from Italy.

**Comment by alessio0292** [comment]
Please create a robot because the collaborators of team talk don't want to do nothing for me!

**Comment by lbk2907** [comment]
Sorry.
I think this is not related to this project.
Thanks.

**Comment by alessio0292** [comment]
Yes, is related because bjoern doesn't nothing for me and the robot can 
do many things.

Il 11/10/2022 10:36, lbk2907 ha scritto:

**Comment by beqabeqa473** [comment]
No one has obligations to do something for you or other people.
You are using a free product and maximum what you can do is to suggest
specific feature and don't talk about that repeatedly.
If some of devs have a resources or desire to do it, they will.

There is no other way.

Of course i think you can buy a product and pay for your personal
requests, in this way, i think you will have more chance to get what
you want

On 10/11/22, alessio0292 ***@***.***> wrote:
> Yes, is related because bjoern doesn't nothing for me and the robot can

> do many things.

>

> Il 11/10/2022 10:36, lbk2907 ha scritto:

>> Sorry.

>> I think this is not related to this project.

>> Thanks.

>>

>> —

>> Reply to this email directly, view it on GitHub

>> <https://github.com/BearWare/TeamTalk5/discussions/1343#discussioncomment-3847512>,
>>

>> or unsubscribe

>> <https://github.com/notifications/unsubscribe-auth/AWJT5PF7OHF7U5QX5VYHE4TWCURIHANCNFSM5XHDQ4NQ>.

>> You are receiving this because you authored the thread.Message ID:

>> ***@***.***>

>>

>

> --

> Reply to this email directly or view it on GitHub:

> https://github.com/BearWare/TeamTalk5/discussions/1343#discussioncomment-3851054

> You are receiving this because you are subscribed to this thread.

>

> Message ID:
> ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Reply by alessio0292** [reply]
I don't like your thing that You said. The robot for me is important because i did many discussions for Bjoern but this person doesn't speak more with me, doesn't want to do nothing for me. I have problem with a creator of team talk I sent also for you a e-mail in my account of hotmail. My address is colizza_alessio@hotmail.it i sent for you this message in august 2022.

**Comment by alessio0292** [comment]
I don't like more a team talk of Bjoern and I'd like a modified team talk if someone can do for me for example Beca, Corentin or others or a robot that i suggest.

**Comment by alessio0292** [comment]
features that i would like in this modified team talk:
1. A possibiliti of create channels for only operators or creators of channels and modify channels for example create permanent channels without modify or delete other channels. A new option.
2. a possibility of join in the servers in total anonimity or invisible mode. Create an invisible status for join in total anonimity in the servers.
3. a possibility of create a only listen mode in me menu for people that don't want or can't speak.
4. A possibility, also for users, of intercept people in other channels.
5. A possibility, again in windows seven of transmit media files and a possibility in seven of transmits links or urls.
6. A possibility of record in mp3 in seven again.
7. A possibility of configure audio codecs also if a person is in a channel or if a person is configuring a initial channel in a server.
8. A possibility of transmit with a minimum interval of 0 millisecond.
For this moment These are the features that i'd like in this modified team talk for example a edited or modified 5.10 or 5.11 qt and classic for seven.

**Comment by alessio0292** [comment]
in the classic version I would like also the option to activate or deactivate the alert when someone is intercepted in another channel, an option to activate the display notifications or deactivate them and an option to choose in the display which notifications to activate or deactivate.
